openreview/conference/templates/areachairWebfield.js

```diff
@@ -149,64 +142,72 @@ var getInvitationRegex = function(name, numberStr) {
 };
 
 // Ajax functions
-var loadData = function(acPapers, secondaryAcPapers) {
-  var noteNumbers = _.concat(acPapers, secondaryAcPapers);
+var loadData = function(paperNums) {
+  var acPapers = paperNums.areaChairPaperNums;
+  var secondaryAcPapers = paperNums.secondaryAreaChairPaperNums;
+  var noteNumbers = _.uniq(_.concat(acPapers, secondaryAcPapers));
   var blindedNotesP;
   var metaReviewsP;
   var allReviewersP;
   var secondaryMetaReviewsP;
+  var unformattedOfficialReviewsP;
 
   if (noteNumbers.length) {
     var noteNumbersStr = noteNumbers.join(',');
 
+    var metaReviews = [];
+    var secondaryMetaReviews = [];
+    var unformattedOfficialReviews = [];
+
     blindedNotesP = Webfield.getAll('/notes', {
       invitation: BLIND_SUBMISSION_ID,
       number: noteNumbersStr,
-      details: 'invitation,replyCount'
+      select: 'id,number,forum,content,details,invitation',
+      details: 'invitation,replyCount,directReplies'
     }).then(function(notes) {
+      _.forEach(notes, function(note) {
+        _.forEach(note.details.directReplies, function(directReply) {
```

Comment:
Using a regular for loop for both of these might actually give a good speedup. The overhead of the lodash forEach function can add up over 10k+ iterations.

Reply:
Done!